SpatialGDK/Source/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -13,6 +13,13 @@ DECLARE_LOG_CATEGORY_EXTERN(LogSpatialWorkerConnection, Log, All);
 DECLARE_DELEGATE(FOnConnectedDelegate);
 DECLARE_DELEGATE_OneParam(FOnConnectFailedDelegate, const FString&);
 
+enum class SpatialConnectionType
+{
+	Receptionist,
+	Locator,
```

Comment:
I think `Locator` should be changed to `LegacyLocator` and `LocatorV2` be changed to just `Locator` as this is how games will be launched going forwards.